tools/feature_matrix.py

```diff
@@ -70,13 +77,18 @@ def caniuse(feature):
   return True
 
 
-def enable_feature(feature):
+def enable_feature(feature, reason):
   """Updates default settings for browser versions such that the given
   feature is available everywhere.
   """
   for name, min_version in min_browser_versions[feature].items():
     name = f'MIN_{name.upper()}_VERSION'
-    if settings[name] < min_version and name not in user_settings:
+    if settings[name] < min_version:
+      # If the user explicitly chose and older version we error out here.
+      # Otherwise we bump the minimum version to accommodate the feature.
+      if name in user_settings:
+        exit_with_error(f'{name}={user_settings[name]} is not compatiable with {reason} '
+                        f'({min_version} or above required)')
```

Comment:
Perhaps we shouldn't error in this case? I can see an argument for just warning, and trusting the user knows what they want otherwise. For example perhaps they have some kind of polyfill or workaround (like they know a bigint-using export will never actually be called).

Reply:
Good point.  Done